admin/environment-management/ssh-access.md

```diff
@@ -1,17 +1,113 @@
 ---
-title: "Disable SSH Access"
-description: Learn how to disable SSH to comply with ingress restrictions.
+title: "SSH Configuration"
+description: Learn how to configure SSH access to Coder environments.
 ---
 
-By default, Coder enables SSH access for all users. There are no additional
-dependencies required.
+By default, Coder enables SSH access for all users. Coder assigns each user a
+private key that they can use to access their environments.
+
+## Background
+
+Part of the standard Coder environment asset bundle is a lightweight SSH server
+mounted onto the environment agent; the lightweight SSH server is a backup used
+when Coder can't find a server available on port 22. This allows slimmer images
+to remain accessible via SSH without the need for additional image dependencies.
+
+## Using OpenSSH
+
+The built-in SSH server is limited and does not correctly implement
```

Comment:
I'm not sure we should say "does not correctly implement" as it sounds like more of a "we know this is wrong" or "a bug" type of thing. Maybe a more positive light on it... something like this (just removing `correctly`)

```suggestion
The built-in SSH server is limited, and does not implement
```